src/Domain/Amr/Tags/NeighborFlags.hpp

```diff
@@ -0,0 +1,26 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <unordered_map>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "Domain/Amr/Flag.hpp"
+
+/// \cond
+template <size_t VolumeDim>
+class ElementId;
+/// \endcond
+
+namespace amr::domain::Tags {
+/// amr::domain::Flag%s for the neighbors of an Element.
+template <size_t VolumeDim>
+struct NeighborFlags : db::SimpleTag {
+  using type = std::unordered_map<ElementId<VolumeDim>,
```

Comment:
Do you need to include the direction?  Maybe not because the data is not associated with an element face.

Reply:
I have not needed the direction so far as these flags are not direction dependent